storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManager.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package kafka.log.remote;
+package org.apache.kafka.server.log.remote.storage;
```

Comment:
Could you update the `metricsGroup` to ensure the metrics is not changed by the new package?

Reply:
Ahh, I keep making this mistake!!! Thanks for pointing it out 👍

Reply:
I pushed a fix